Enable LLD_REPORT_UNDEFINED by default (#16003)
emscripten-core/emscripten: ChangeLog.md

3.1.28 (in development)
-----------------------
- `LLD_REPORT_UNDEFINED` is now enabled by default.  This makes undefined symbol
  errors more precise by including the name of the object that references the

>   undefined symbol. The old behaviour (or allowing all undefined symbols are

Reviewer: ```suggestion
  undefined symbol. The old behaviour (of allowing all undefined symbols at
```
